### ISSUE TITLE
`Toggle Diff Editor` should have an assigned keybinding

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#238145]
1. enable a screen reader
2. create some file edits - one that removes something from the file, one that adds something
3. 🐛 cannot review the removed changes because cannot navigate them with one's cursor
4. 🐛 cannot know where the modified lines end because there are no audio cues
5. one workaround would be to access the diff view for the edits. Currently, there's no keybinding to do so and no way to assign one AFAICT. 

If we had that, a user could at least review the diff, navigate back (or ideally use the same keybinding to toggle back to the edits editor position corresponding to the diff location), and accept/discard changes from there.

We will also eventually want to either:
- enable screen reader navigation of the edits editor by adding audio cues and support for traversing lines in the deleted code sections
- open the diff view by default for screen reader users and add the inline actions of accept/discard, navigate between edit changes

https://github.com/user-attachments/assets/15250aba-dc4c-4a03-a590-77fdcb68a15d

cc @isidorn

**Comment by isidorn** [comment]
Thanks for opening this. I agree that we need to make it possible for screen reader users to review and accept and reject the changes.

Thus having commands for accept/reject based on cursor position sounds like a logical next step.
And in addition the user needs to be aware what is actually added / removed. One option is to open a diff view as you propose.

**Comment by meganrogge** [comment]
@jooyoungseo suggested that `f7` would be great as it would align with our other diff experiences.
